Hearthstone/app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background"
+    android:orientation="vertical"
+    tools:context=".ui.HomePage.HomeFragment">
+
+
+    <ImageView
+        android:id="@+id/emblem"
+        android:layout_width="0dp"
+        android:layout_height="140dp"
+        android:layout_marginStart="23dp"
+        android:layout_marginTop="36dp"
+        android:layout_marginEnd="23dp"
```

Comment:
You should start adding your dimensions to the dimens file instead of hard coding them.